xapian-core/include/xapian/query.h

```diff
@@ -219,6 +227,22 @@ class XAPIAN_VISIBILITY_DEFAULT Query {
 	 */
 	OP_VALUE_LE = 12,
 
+    /** Match only documents where a value slot is < a given value.
+	 *
+	 *  Similar to @a OP_VALUE_RANGE, but open-ended.
+	 *
+	 *  This operator never contributes weight.
+	 */
+	OP_VALUE_LT = 18,
+
+    /** Match only documents where a value slot is = a given value.
+	 *
+	 *  Similar to @a OP_VALUE_RANGE, but open-ended.
+	 *
+	 *  This operator never contributes weight.
+	 */
+	OP_VALUE_EQ = 19,
```

Comment:
As noted, you probably want to remove this. Note also that the doccomment is incorrect: this is not an open-ended operator.